using_openshift/port_forwarding.adoc

```diff
@@ -0,0 +1,109 @@
+= Port Forwarding
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+You can use the CLI to forward one or more local ports to a pod. This allows you to listen on a given or random port locally, and have data forwarded to and from given ports in the pod.
+
+== Basic Usage
+Support for port forwarding is built into
+link:../using_openshift/cli.html#common-cli-operations[the CLI]:
+
+****
+`$ osc port-forward -p _<Pod>_ _[<Local_Port>:]<Pod_Port>_ _[[<Local_Port>:]<Pod_Port> ...]_`
+****
+
+The CLI listens on each local port specified by the user, forwarding via the link:#protocol[protocol] described below.
+
+Ports may be specified using the following formats:
+
+`5000`::
+  The client listens on port 5000 locally and forwards to 5000 in the
+pod.
+`6000:5000`::
+  The client listens on port 6000 locally and forwards to 5000
+in the pod.
+`:5000` or `0:5000`::
+  The client selects a free local port and
+forwards to 5000 in the pod.
+
+For example, to listen on ports `5000` and `6000` locally and forward data to and from ports `5000` and `6000` in the pod, run:
+
+====
+
+----
+$ osc port-forward -p mypod 5000 6000
+----
+====
+
+To listen on port `8888` locally and forward to `5000` in the pod, run:
+
+====
+
+----
+$ osc port-forward -p mypod 8888:5000
+----
+====
+
+To listen on a free port locally and forward to `5000` in the pod, run:
+
+====
+
+----
+$ osc port-forward -p mypod :5000
+----
+====
+
+Or, alternatively:
+
+====
+
+----
+$ osc port-forward -p mypod 0:5000
+----
+====
+
+== Protocol
+Clients initiate port forwarding to a pod by issuing a request to the
+Kubernetes API server:
+
+****
+`/proxy/minions/_<Node_Name>_/portForward/_<Namespace>_/_<Pod>_`
```

Comment:
This URL will most likely change before GA, assuming we get our bastion/proxy implemented. cc @csrwng